tst/LargeAlgebras.tst

```diff
@@ -0,0 +1,83 @@
+gap> START_TEST( "LargeAlgebras.tst");
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236000));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 60 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[ [ infinity, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236001));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 0, 0 ], [ 60, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236002));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 60 ], [ 2, 29, 0 ], [ 2, 31, 0 ] ], [ [ 60, 0, 60 ], [ 0, 60 ], [ 0 ] ] 
+ ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236003));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 1, NF(5,[ 1, 4 ]), 120, [ [ 2, 11, 0 ], [ 2, 19, 0 ], [ 2, 29, 60 ], [ 2, 31, 0 ] ], [ [ 0, 60, 0 ], [ 0, 0 ], [ 0 ] ] ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);               
+[ [ 5, 2 ] ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236004));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 8, NF(120,[ 1, 79, 91, 109 ]) ]
+
+#
+gap> QG:=GroupRing(Rationals,SmallGroup(1920,236005));;
+gap> W:=WedderburnDecompositionInfo(QG);;
+gap> A:=W[Length(W)];
+[ 8, NF(120,[ 1, 29, 91, 119 ]) ]
+gap> LocalIndicesOfCyclotomicAlgebra(A);
+[  ]
+
+#
+QG:=GroupRing(Rationals,SmallGroup(1920,236006));;
```

Comment:
There is a missing `gap>` prompt before `QG`, hence it is wrongly recognised as output. You can amend the commit by updating the file, saving changes, and then calling
```
git add tst/LargeAlgebras.tst 
git commit --amend
git push angeldelriomateos -f
```